src/components/TicketModal/index.jsx

```diff
@@ -28,7 +40,7 @@ export const TicketModal = (props) => {
             <Card.Content>
               <div className="card__summary">
                 <div className="close-icon-container">
-                  <button type="button" onClick={props.onClose}>
+                  <button type="button" onClick={props.handleCloseTicket}>
```

Comment:
"handleCloseTicket" is not a **prop** given to this modal, just a method defined above, so this should just be 
```onClick={handleCloseTicket}```